packages/bruno-app/src/components/Modal/index.js

```diff
@@ -97,6 +97,11 @@ const Modal = ({
   if (hideFooter) {
     classes += ' modal-footer-none';
   }
+
+  useEffect(() => {
+    document.getElementsByTagName('dialog')[0].showModal();
```

Comment:
I think `useRef` should be used here instead of accessing the DOM directly.

Reply:
https://github.com/usebruno/bruno/pull/2076/commits/d467d837efb0a4390ee35a12d85da8690eb9a70d